portfolio/mbosquezgonzalez-sps-summer21-151f1a946d04.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "mbosquezgonzalez-sps-summer21",
+  "private_key_id": "151f1a946d04e417a32fc743071a238247669839",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCsMDWjcwHyfS0D\nheaYIZlMwtmz624dIUtHmUdxy0oHtiKTML6GT8SEtWQkbe+wTeQ2+35Rsp2AJOao\npM4iXPJ9Sloc3X5CDhppVgTeuXCvf0l1MxKVaVE62y+4YI3rRSWBUx6NEFLVvhul\nFt07GYV2aXZdDkyq7b8/tcXCPI23SLPdKBF8+V1bViP04Bw+zBY0/JAXyEiL0Dhj\ns9ZWSzEPxETdReF8WK4JvgeXaPRY9FO/Mgbu7M1i97NmPdnK09j3ncML/m7uXHY9\nPdRUT8f0WtdIhqkIc/tjquxkSR8XSVXaIWRYjG1Z8h/EaTqNytfGj1Zk5pBUmb1o\n8i66DvHbAgMBAAECggEALJUE5k+KQwcvBvj95gjZANt4c5Gy/n7W13Bz454/ijSZ\nyiN2SR9RIg03IwmF5E+ela/TGoMVLY8gW6zkJFiNfmwoXc3peV/ebcSUWTiJWR8r\nXVl+Z8oPep4ywlnZlP9fwRbM07hUwjvJO5rqZ/lpKWoP9/MSXbVV7cB6NhrolHol\nGZ6S0d5q1BrDIqtfaHfiXBVfUvAicFUxMXBaJR8F+p54i6bAILudbeCyWSHF6N+4\nwqY5v4km4sZzNC3JlIIr4mJxMIIqXXPCfZYLK1wQmeG7GBDzwU7FTrXAqxSLcIze\nnW3VSt5rqhlODJiqsRw0qvn+AsyoXSdB3zX2CaJn/QKBgQDdu/5SnPK3N+FZwp0C\nk6/RCiHccSi07UQotPxX8sOEVmKRVANaOAtx8CzLYoSFz+UWB9xOjBUhqouXlaUn\nRbYkk9LO4/0fj2tk0gYBH1y7cUqseUKePqfcPAMoKowFeG/fHPNdXHzOE+WSJqY5\nEx5goGvZA0XfUIOBKLq8Dq4phQKBgQDGzCGhPO1BRJm551VgcWBf3xmfYInQ2Xle\nTDbqVciL3O1GL3OMdPfNrgq9nbS0Qbg/qg8GcO9doeCpcKUKvjVJckN9ChM8K9F4\nOuEnuXP7J8pp82upp9oJmo4qhDQsE+cypdlPylTwF9x/44b8eUebKnlCH6aSpi1o\nhBRWVcTb3wKBgQDQStUxzfXDX1717veHgCzc9eOxLSvMKkao5rtV8trv6iZV1tH/\nYj5+Hdw2gqly+kR9WYz+zGDda1QOZajrJ3VfDu4tI8aYayW7Y//39foqqtY3BwIp\nnToajNV6Wd2HFgghUxTaUxN6kv5JOL1VSDx1EL2+8nAkw4GFRzmDh1bb8QKBgGDP\nSdVGYVe2e5I/qKEVtVQaxlSJ47NUJLmaLegCLG6IAPh75S5SS9ba2ZorcbOSiZJf\nU3/UgYy8Zza8h3Lq2/JPdbQvstxLjm+ZZMo+ks+NOhJMm3J8Lj8dB/yGcw3AB4fv\nohssMh7Vz+hrufG7cfaLQY/E8A5cZKKQJ4M/s2VLAoGAMWTTCsuksBe7tCEpJfxq\nfNhKatXsTEfYoU8GlK5IsFZWBl2aaUUg3jox97y08Qw0qf3FvW0Sq1zWAsDvJLzw\noLVledYuoa2OIdGeOuAbwEENGsPvRiCkZo6oSEXdwXEgjB46va866luyydmXXrLY\nUXnytvnW/Q7DKbohCdMCA7A=\n-----END PRIVATE KEY-----\n",
```

Comment:
This looks like something that actually shouldn't be checked into source control. Other people (or bots) could see it here and use it to impersonate your app. I'd recommend removing this file from git's tracking (or keeping any sensitive info in a different file, and *regenerating your private key*. People often think it's enough to just take the info down from Github, but really once it's exposed it's compromised. This probably isn't a huge deal here for this API but there are other situations in which exposing a private key could be a big security vulnerability.